### PR TITLE
feat(odyssey-react): add Box to Tabs components

### PR DESCRIPTION
**Pull request by andrewberg-okta**

### CODE REVIEW

File: packages/odyssey-react/src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -163,15 +164,15 @@ export const Tabs = withTheme(
 
 const Container = forwardRef<HTMLDivElement, PropsTabsContainer>(
   ({ children, id, ariaLabel }, ref) => (
-    <div id={id} aria-label={ariaLabel} data-testid="ods-tabs" ref={ref}>
+    <Box id={id} aria-label={ariaLabel} data-testid="ods-tabs" ref={ref}>
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Tabs/Tabs.tsx:167]: 😞 forgot all about the hardcoded testid value here. We should remove this

File: packages/odyssey-react/src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -196,26 +197,30 @@ const Tab = function TabsTab({
...
       id={id}
       role="tab"
       tabIndex={selected ? 0 : -1}
       aria-controls={ariaControls}
       aria-selected={selected}
       className={componentClass}
       onClick={onClick}
+      color={false}
+      fontSize={false}
+      fontWeight={false}
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Tabs/Tabs.tsx:211]: similar question to other PRs, how are you determining which properties need to be omitted here from the Box default styles?

**Review by jeffbelser-okta — APPROVED** [review]

File: packages/odyssey-react/src/components/Tabs/Tabs.tsx (hunk 2 above)
Comment (andrewberg-okta) [reply packages/odyssey-react/src/components/Tabs/Tabs.tsx:211]: anything that is set by the component root element style needs to be opted out of or it gets overridden by Box and additionally cannot be themed.

The `.tab` style sets color, size, and weight
https://github.com/okta/odyssey/blob/d4bfae9c94c4906d4b67f085cb808939f9014126/packages/odyssey-react/src/components/Tabs/Tabs.module.scss#L18

Comment (jeffbelser-okta) [reply packages/odyssey-react/src/components/Tabs/Tabs.tsx:211]: got it thanks that aligns with my understanding. Regarding theming I am going to create a JIRA to follow up on this batch of work so we can finalize the effort

**Pull request merged**